src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,6 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return (a != b && c != d) || (a != c && b != d);
```

Comment:
Good job, try to simplify it with binary operators

Reply:
done